### ISSUE TITLE
gapi.load() clears the console.

### ISSUE THREAD

**Issue by mesqueeb** [google/google-api-javascript-client#394]
I have a weird bug where as soon as I do `gapi.load('client:auth2')` it will clear out the console.
This makes it hard to debug stuff that happen before gapi.load.

Is this the normal behaviour?

**Comment by TMSCH** [comment]
@mesqueeb Is it on Chrome? If so, this is an issue with Chrome itself not the library. Adding an IFrame to the DOM does that in newer versions of Chrome. You can click the option to Preserve Log to avoid the console being cleared.

**Comment by mesqueeb** [comment]
@TMSCH 
Thank you very much. Preserve Log in the console does keep the logs.
Now I noticed the behaviour that when I open my app it says:
> Navigated to https://accounts.google.com/o/oauth2/postmessageRelay?parent=http%3A%2F%2Flocalhost%3A5000&etc........ (omitted)

Is this behaviour normal? (If so you can just thumbs up this post)  : D

**Comment by TMSCH** [comment]
It is yes.

**Comment by misterbridge** [comment]
Sorry to unburry this thread but you say it's a Chrome bug, is there anywere I can find mention on this ?
I couldn't find anyone talking about this bug until this thread, I'm at least glad I'm not the only one.
I just can't find the proper words to search this bug online ... :(

**Comment by misterbridge** [comment]
Alright found it

TL;DR: Wait for chrome 66

https://bugs.chromium.org/p/chromium/issues/detail?id=823894
https://bugs.chromium.org/p/chromium/issues/detail?id=818209
